backend/CSPEC_Parser.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+#CSPEC_Parser.py
+#----------------------------------
+# Created By : Matthew Kastl
+#----------------------------------
+""" The CSPEC or chart specification file is a file containing all instruction and information
+for Flare to run a generate a data CSV. This file contains a parent parser that will execute sub parsers
+for any version of CSPEC.
+ """ 
+#----------------------------------
+# 
+#
+#Imports
+from os.path import exists
+from json import load
+from DataClasses import *
+
+class CSPEC_Parser:
+    def __init__(self, file_path: str) -> None:
+
+        if not exists(file_path):
+            print(f'{file_path} not found!')
+            raise FileNotFoundError
+        with open(file_path) as CSPEC_file:
+            
```

Comment:
wild space to have an extra newline to me but this is super nit picky

Reply:
I dont like putting comments right after like iteration lines. I fixed it to be the best of both worlds! STYLING IS IMPORTANT!